### PR TITLE
Update pipeline release v0.60.x

### PR DESCRIPTION
**Pull request by JeromeJu**
# Changes
This commit updates the pipeline release v0.60.x 

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/website/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: sync/config/pipelines.yaml [hunk 1]
```diff
@@ -26,6 +26,16 @@ archive: https://github.com/tektoncd/pipeline/tags
 #       header: <dict>         # optional, no header added if not set
 #         See https://www.docsy.dev/docs/adding-content/navigation/#section-menu
 tags:
+- name: release-v0.60.x
+  displayName: v0.60.x-LTS
```
Comment (AlanGreene) [inline sync/config/pipelines.yaml:30]: I don't think v0.60 is an LTS release.
The previous release, v0.59 last month, was an LTS release.

Comment (AlanGreene) [reply sync/config/pipelines.yaml:30]: We should still add the docs for v0.60, just without the LTS label in the displayName

Comment (JeromeJu) [reply sync/config/pipelines.yaml]: Thanks @AlanGreene  updated.

**Review by AlanGreene — APPROVED** [review]
/lgtm

**Pull request merged**